### ISSUE TITLE
fail config with SKIP_SPIRV_TOOLS_INSTALL=ON

### ISSUE THREAD

**Issue by sl1pkn07** [KhronosGroup/SPIRV-Tools#3123]
when try build/install with `SKIP_SPIRV_TOOLS_INSTALL=ON`, gets

~~~
CMake Error: install(EXPORT "SPIRVTargets" ...) includes target "SPIRV" which requires target "SPIRV-Tools-opt" that is not in any export set.
~~~

**Comment by s-perron** [comment]
What do you expect to happen if you are skipping installation of spirv tools?  It looks this is the cmake command for glslang.  You build with spirv-tools on, but then you cannot install glslang because it depends on spirv-tools. 

That seems to be what it happening, and seem perfectly reasonable.  Am I missing something?

**Comment by sl1pkn07** [comment]
yes, the option is set in spirv-tools but appears when build througth glslang. and working in the past (my last build with the option working is in sáb sep 14 12:49:35 CEST 2019, vulkan-loader commit 0044fcf11)


the main problem is when you try to build the projects in 32bits mode and you not need install the excutables (but yes the libs), because you have builded the programs (and libs) in the 64 bits mode. because make conflicts between 32 and 64 bits executable (name colisions)

**Comment by sl1pkn07** [comment]
Also, seems 'SKIP_SPIRV_TOOLS_INSTALL', i think should work when try to do 'make install' (i mean, do no install the spirv-tools progs, libs and headers), not when do configure or build the project

the 'INSTALL' word in the option can mislead
